src/main/java/com/codeborne/selenide/ScrollIntoViewOptions.java

```diff
@@ -0,0 +1,55 @@
+package com.codeborne.selenide;
+
+import static com.codeborne.selenide.ScrollIntoViewOptions.Behavior.instant;
+import static com.codeborne.selenide.ScrollIntoViewOptions.Block.start;
+import static com.codeborne.selenide.ScrollIntoViewOptions.Inline.nearest;
+
+/**
+ * See <a href="https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView">scrollIntoView documentation</a>
+ *
+ * @param behavior whether scrolling is instant or animates smoothly
+ * @param block the vertical alignment of the element within the scrollable ancestor container
+ * @param inline the horizontal alignment of the element within the scrollable ancestor container
+ */
+public record ScrollIntoViewOptions(
+  Behavior behavior,
+  Block block,
+  Inline inline
+) {
+
+  public static ScrollIntoViewOptions instant() {
```

Comment:
Maybe it's better to call it `default` and fullfill with default values based on documentation: 

- behavior: null (or auto)
- block: start
- inline: nearest

Give an ability to set `behavior` also. And then override values for `scrollTo` and `scrollIntoView`methods.

Reply:
Usually, it would be reasonable.
But I wanted to force using "instant" because it avoids risk of flaky tests.

The default value "auto" can scroll smoothly (slowly) which often causes random test failures.

Reply:
Maybe.
But it's still possible to set behaviour in constructor: `new ScrollIntoViewOptions(auto, block, inline)`.